tests/require-passive-events.js

```diff
@@ -47,7 +47,7 @@ ruleTester.run('require-passive-events', rule, {
       ]
     },
     {
-      // Intentionally mispelled!
+      // Intentionally misspelled!
```

Comment:
:)

Reply:
Yeah, I didn't correct the intentional typo below :)